src/components/hub/elements/MessageInput.js

```diff
@@ -82,7 +83,7 @@ export default function MessageInput({
   }, [location]);
 
   const inputsView = useMemo(() => {
-    if (isBlockedConv) {
+    if (isBlockedConv || isEncryptedSessionActive === false) {
```

Comment:
`isEncryptedSessionActive === false` ->  `!isEncryptedSessionActive`

Reply:
can`t do this because the null value also has its own meaning